radio/src/gui/Taranis/menu_model_setup.cpp

```diff
@@ -570,9 +588,40 @@ void menuModelSetup(uint8_t event)
         if (attr) g_model.noGlobalFunctions = !checkIncDecModel(event, !g_model.noGlobalFunctions, 0, 1);
         break;
 
-      case ITEM_MODEL_INTERNAL_MODULE_LABEL:
-        lcd_putsLeft(y, TR_INTERNALRF);
+       case ITEM_MODEL_INTERNAL_MODULE_LABEL:
```

Comment:
The code below is impossible to read, perhaps you use tabs with tab != 8 chars which renders very badly (non standard). We use 2 chars in this project.